### PR TITLE
Fix url loss after the save online operation

### PR DESCRIPTION
**Pull request by vpereverzev**
Prevent the loss of the user's score URL by automatically saving the local score file when receiving the URL from the web server during the “Save Online” operation.

We found out that some users lost links to their scores after they saved the score online.

It turned out that the current process requires from the user not quite obvious additional steps to save:

- the user saves his score locally
- the user clicks the "Save online" button
- the editor receives a new link to the score and modifies the .mscz file, adding a meta tag with a link to the score
- the file is modified and the user must save it again locally so as not to lose the link. Otherwise, the next time he goes to the editor and opens his score locally, there will be no link in it. So the next time he'll try to save online, duplicates will form on the server-side.

### CODE REVIEW

File: libmscore/scorefile.cpp [hunk 1]
```diff
@@ -512,6 +512,15 @@ bool MasterScore::saveFile(bool generateBackup)
       return true;
       }
 
+//---------------------------------------------------------
+//    isFileExisting
+//---------------------------------------------------------
+
+bool MasterScore::isFileExisting() const
```
Comment (igorkorsukov) [inline libmscore/scorefile.cpp:519]: There is a `_created` flag, probably it should be used

**Comment by anatoly-os** [comment]
Looks good. Need thorough testing though to verify whether the solution works well on edge cases.

File: libmscore/scorefile.cpp (hunk 1 above)
Comment (vpereverzev) [reply libmscore/scorefile.cpp]: yep, didn't notice it

**Comment by Jojo-Schmitz** [comment]
Indeed a pretty annoying issue which comes up in the forum time and time again (like the need to copy/paste the URL from musescore.com into the source filed of File > Score properties)

**Comment by vpereverzev** [comment]
> Looks good. Need thorough testing though to verify whether the solution works well on edge cases.

@anatoly-os yep, i've tested both cases:
- save online generated file (without local copy)
- save online existing file (with local copy)
